pkg/repository/storage/s3/s3upload.go

```diff
@@ -64,3 +70,48 @@ func UploadImage(file io.ReadCloser, filename string) (string, error) {
 
 	return filepath, nil
 }
+
+// This checks if an s3 bucket key exists
+func keyExists(key string) (bool, error) {
+	svc := s3.New(s3session)
+	MyBucket = config.GetConfig().S3.BucketName
+
+	_, err := svc.HeadObject(&s3.HeadObjectInput{
+		Bucket: aws.String(MyBucket),
+		Key:    aws.String(key),
+	})
+
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
```

Comment:
you should use **err** always for any error
no need to have these weird variable names
once an **err** is handled is should be used for other errors

Reply:
okay, my bad :sweat: